storops/unity/resource/system.py

```diff
@@ -54,9 +58,40 @@ def get_sp(self, _id=None, name=None, **filters):
         return self._get_unity_rsc(UnityStorageProcessorList, _id=_id,
                                    name=name, **filters)
 
+    def get_iscsi_portal(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(UnityIscsiPortalList, _id=_id,
+                                   name=name, **filters)
+
+    def get_ethernet_port(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(UnityEthernetPortList, _id=_id,
+                                   name=name, **filters)
+
+    def get_host(self, _id=None, name=None, **filters):
+        return self._get_unity_rsc(UnityHostList, _id=_id,
+                                   name=name, **filters)
+
+    def get_initiator(self, _id=None, **filters):
+        return self._get_unity_rsc(UnityHostInitiatorList, _id=_id, **filters)
+
     def get_lun(self, _id=None, name=None, **filters):
         return self._get_unity_rsc(UnityLunList, _id=_id, name=name, **filters)
 
+    def create_lun(self, name, pool, size_gb, sp=None, host_access=None,
+                   is_thin=None, description=None, tiering_policy=None,
+                   is_repl_dst=None, snap_schedule=None, iolimit_policy=None):
+
+        if sp is None:

```

Comment:
No need to check `None`.  I think you already give a default value in `UnityLun.create`.